askama_shared/src/parser.rs

```diff
@@ -144,7 +144,15 @@ pub enum Target<'a> {
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub struct Ws(pub bool, pub bool);
 
-pub type Cond<'a> = (Ws, Option<Expr<'a>>, Vec<Node<'a>>);
+pub type Cond<'a> = (Ws, Option<CondTest<'a>>, Vec<Node<'a>>);
+
+type CondTest<'a> = (
```

Comment:
Let's make this a `struct` with named fields, I guess the first field could be named `target` or `pattern`?